FirebaseSharedSwift/Sources/third_party/FirebaseDataEncoder/FirebaseDataEncoder.swift

```diff
@@ -2611,11 +2611,14 @@ fileprivate struct _JSONKey : CodingKey {
 //===----------------------------------------------------------------------===//
 
 // NOTE: This value is implicitly lazy and _must_ be lazy. We're compiled against the latest SDK (w/ ISO8601DateFormatter), but linked against whichever Foundation the user has. ISO8601DateFormatter might not exist, so we better not hit this code path on an older OS.
-fileprivate var _iso8601Formatter: ISO8601DateFormatter = {
-  let formatter = ISO8601DateFormatter()
-  formatter.formatOptions = .withInternetDateTime
-  return formatter
-}()
+fileprivate var _iso8601Formatter: ISO8601DateFormatter {
```

Comment:
formatters may be expensive. Does it work to make line 2614 a `let` instead of a `var`?

Reply:
good catch, I did see some posts that they are. If I make it a let:

```
Let '_iso8601Formatter' is not concurrency-safe because non-'Sendable' type 'ISO8601DateFormatter' may have shared mutable state
```

Reply:
I see two options:
1. use `nonisolated(unsafe)` to silence
1. wrap the formatter in an unchecked Sendable type

Since both are doing silencing rather than fixing, I prefer 1 since it's less LOC.